src/database/mixxxdb.cpp

```diff
@@ -53,7 +53,7 @@ mixxx::DbConnection::Params dbConnectionParams(
     // See also:
     // https://www.sqlite.org/inmemorydb.html
     if (inMemoryConnection) {
-        params.filePath += QStringLiteral("?mode=memory&cache=shared");
+        params.filePath += QLatin1String("?mode=memory&cache=shared");
```

Comment:
The QStringLiteral is preferred for concatenation according to the link @daschuer shared.

Reply:
Weird that the Qt blog says the opposite. I'll revert that.

Reply:
The Qt blog does not mention the different types for concatenation.

In the long term we should rely on the clazy hints and otherwise just use the representation that requires less boilerplate. Prefer simple rules.